src/components/AddForm.js

```diff
@@ -6,28 +6,41 @@ import {
   addTripsToLocalStorage,
   getTripsFromLocalStorage,
 } from "../services/tripsStorage";
+import axios from "axios";
 
 function AddForm() {
   const [destinationInput, setDestinationInput] = useState("");
   const [activitiesInput, setActivitiesInput] = useState("");
   const [restaurantsInput, setRestaurantsInput] = useState("");
   const [notesInput, setNotesInput] = useState("");
+  const [imageUpload, setImageUpload] = useState("");
 
   function handleSubmit(e) {
     e.preventDefault();
-    addTripsToLocalStorage({
-      id: `${destinationInput}`,
-      destination: destinationInput,
-      activities: activitiesInput,
-      restaurants: restaurantsInput,
-      notes: notesInput,
-    });
-    const trips = getTripsFromLocalStorage();
-    console.log(trips);
-    setDestinationInput("");
-    setActivitiesInput("");
-    setRestaurantsInput("");
-    setNotesInput("");
+    const formData = new FormData();
+    formData.append("file", imageUpload);
+    formData.append("upload_preset", "tyikvr8a");
+
+    axios
+      .post("https://api.cloudinary.com/v1_1/dyjecx1wm/image/upload", formData)
+      .then((response) => {
+        const imageURL = String(response.data.secure_url);
+        addTripsToLocalStorage({
+          id: `${destinationInput}`,
```

Comment:
is it necessary to have the template literals here? you are not using them in the line below for the same variable

Reply:
you're absolutely right! I'll remove it :)